app/models/work_version.rb

```diff
@@ -106,6 +106,10 @@ def draft?
     version_draft? || first_draft?
   end
 
+  def deleteable?
+    (first_draft? || purl_reservation?) || (version == 1 && (pending_approval? || rejected?))
+  end
+
```

Comment:
this could go on the `WorkPolicy` too but it's unclear to me how this would get accessed from the `ButtonsComponent` if so

Reply:
Are the first set of parentheses necessary? If I'm reading correctly, there are 3 OR conditions.